### ISSUE TITLE
Dinamyc table name

### ISSUE THREAD

**Issue by fulltic** [dfahlander/Dexie.js#376]
One question, how can I make the table name dynamic? I have tried db.table ( "table-name"). Get (id) ...

Ex.
Var tableName="customers"
I need db.table(tableName).get(id) OR db.table(tableName).add(obj) ,.... for example, is possible???

Thank you very much

**Comment by dfahlander** [comment]
Yes you can call db.table(tableName). It's all in the docs. The list of tables that you can access is defined in the schema in db.version(x).stores() and those cant just change without closing the database, reopening it and redefine the schema. Please read the docs and samples.

**Comment by disarticulate** [comment]
I just tried in 2.0.0-beta to setup a dyanmic lazy-load for tables and kept getting an ErrorNotFound. I was a little drunk so I spent a few hours trying to figure out what it was doing, and it's not clear that after the initial setup, that new tables will be created. the basic rewrite/reload was:
```
db = null
db = new Dexie('db')
Object.assign(schemas, newschemas)
  db
    .version(dbVersion)
    .stores(schemas) 
```
I even tried mocking and upgrade. As far as I can tell, this code was running prior to trying to put on the database, and I kept getting NotFoundError.
